src/router/Router.jsx

```diff
@@ -7,6 +7,8 @@ import HistoryPage from "../pages/HistoryPage"
 import VideoListingPage from "../pages/VideoListingPage"
 import WatchLaterPage from "../pages/WatchLaterPage"
 import AuthPage from "../pages/AuthPage"
+import LikedVideosPage from "../pages/LikedVideos"
+import PlaylistPage from "../pages/PlaylistPage"
```

Comment:
this is also correct. just a suggestion you can try barrel export :-  https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

Reply:
sure thanks